util/linux_install.sh

```diff
@@ -213,4 +214,4 @@ else
 fi
 
 # Global install tasks
-pip3 install -r ${util_dir}/../requirements.txt
+sudo pip3 install -r ${util_dir}/../requirements.txt
```

Comment:
[What are the risks of running 'sudo pip'?](https://stackoverflow.com/questions/21055859/what-are-the-risks-of-running-sudo-pip)

This is not a good idea. A better solution might be to use `--user` explicitly. It seems that is only sometimes the default.